src/threadpool.c

```diff
@@ -206,7 +225,7 @@ static void init_threads(void) {
     nthreads = MAX_THREADPOOL_SIZE;
 
   threads = default_threads;
-  if (nthreads > ARRAY_SIZE(default_threads)) {
+  if (nthreads != ARRAY_SIZE(default_threads)) {
```

Comment:
Why allocate memory if we need fewer threads than the default?

Reply:
Yeah. Right, that was from a different experiment